src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -107,6 +116,48 @@ function FilterSidebarSearchTab({
         );
     })();
 
+    const styles = {
+        tooltip: {
+            fontSize: '18px',
+            padding: '10px',
+            lineHeight: '22px',
+        },
+        tooltipLineItem: {
+            marginBottom: '6px',
+        },
+        tooltipHeading: {
+            fontWeight: 'bold',
+        },
+    };
+
+    const tooltipTitle = (
+        <div style={styles.tooltip}>
+            <p style={styles.tooltipHeading}>
+                Do you want to see only facilities which these contributors
+                share? If so, tick this box.
+            </p>
+            <p>There are now two ways to filter a Contributor search on the OAR:</p>
+            <ol>
+                <li style={styles.tooltipLineItem}>
+                    You can search for all the facilities of multiple
+                    contributors. this means that the results would show all of
+                    the facilities contributed to the oar by, for example, brac
+                    university or clarks. some facilities might have been
+                    contributed by brac university but not by clarks, or
```

Comment:
This paragraph seems to be missing proper casing

the oar → the OAR
brac university → BRAC University
clarks → Clarks

Reply:
Good catch. I pushed fixup commit b28bad8 to resolve this.